IntelliTect.Multitool/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Security.Claims;
 
-namespace IntelliTect.Multitool.Security;
+namespace IntelliTect.Multitool.Extensions;
```

Comment:
Why move this from Security to Extensions? In general, I prefer to avoid "junk drawer" namespaces (those that contain things that are not related).